backend/models/time_log.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (sequelize, Sequelize) => {
+  const TimeLog = sequelize.define('time_log', {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    date: {
+      type: Sequelize.DATE,
+      allowNull: false
+    },
+    minutes: {
+      type:Sequelize.INTEGER,
+      allowNull: false
+    },
+    description: {
+      type:Sequelize.TEXT,
+      allowNull: false
+    },
+    created_at: {
+      type:Sequelize.DATE,
+      allowNull: false
+    },
+    updated_at: {
+      type:Sequelize.DATE,
+      allowNull: false
+    }
+  },
+  {
+    underscored: true,
+    timestamps: true,
+    createdAt: 'created_at',
+    updatedAt: 'updated_at'
+  })
+
+  return TimeLog
+}
```

Comment:
Add newline at end of file

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b